requirements.txt

```diff
@@ -2,6 +2,7 @@ alabaster>=0.7.12,<1
 matplotlib>=3.1.0,<4
 networkx>=2.3,<3
 numpy>=1.16.1,<2
+pandas==0.24.2
```

Comment:
I used pandas because it was easy to order columns with it. not so easy with csv, so that I can test answers independently from the order of fields in the header.
I did not find such a straightaway function with csv.
I ll add the comment